project_restorer/src/main/scala/PlutoCoreMessageProcessor.scala

```diff
@@ -64,7 +64,7 @@ class PlutoCoreMessageProcessor(mxsConfig:MatrixStoreConfig)(implicit mat:Materi
     updateMessage.status match {
       case status if statusesMediaNotRequired.contains(status)  =>
         Future.sequence(Seq(getNearlineResults(updateMessage.id), getOnlineResults(updateMessage.id)))
-          .map(allResults => processResults(allResults, routingKey, framework, updateMessage.id, updateMessage.status))
+          .map(allResults => processResults(allResults, RoutingKeys.MediaNotRequired, framework, updateMessage.id, updateMessage.status))
```

Comment:
Since `routingKey` is no longer used, it would be nice to remove it from `handleUpdateMessage`'s signature.

Reply:
👍